pkg/ui/src/views/statements/statementDetails.tsx

```diff
@@ -224,13 +225,20 @@ class StatementDetails extends React.Component<StatementDetailsProps, StatementD
     const { parseBarChart, planBarChart, runBarChart, overheadBarChart, overallBarChart } = latencyBreakdown(this.props.statement);
 
     const statsByNode = this.props.statement.byNode;
+    const logicalPlan = stats.sensitive_info && stats.sensitive_info.most_recent_plan_description;
 
     return (
       <div className="content l-columns">
         <div className="l-columns__left">
           <section className="section section--heading">
             <SqlBox value={ statement } />
           </section>
+          <section className="section">
+            <h3>Logical Plan</h3>
+            {logicalPlan
+            ? <PlanView plan={logicalPlan} />
+            : <p>No plan captured yet.</p>}
```

Comment:
Nit: I'd indent the lines starting with `?` and `:`.

But also I've been using this formatter (https://prettier.io/) for the management console and it's saved me a lot of time (and debates like this). Maybe it's time to apply it to the whole codebase (in a separate PR), then have you and @couchand use that going forward.